package-lock.json

```diff
@@ -8039,6 +8039,15 @@
         "websocket-driver": "^0.7.4"
       }
     },
+    "node_modules/sockjs/node_modules/uuid": {
+      "version": "8.3.2",
+      "resolved": "https://registry.npmjs.org/uuid/-/uuid-8.3.2.tgz",
+      "integrity": "sha512-+NYs2QeMWy+GWFOEm9xnn6HCDp0l7QBD7ml8zLUmJ+93Q5NF0NocErnwkTkXVFNiX3/fpC6afS8Dhb/gz7R7eg==",
+      "dev": true,
+      "bin": {
+        "uuid": "dist/bin/uuid"
+      }
+    },
```

Comment:
Make sure you re-run `npm install` or remove `uuid` via `npm uninstall` to remove the dependency from the lockfile too.

Reply:
Done!

Reply:
This was already there, it's part of another dependency

Reply:
If we're not updating packge.json file then there shouldn't be any changes to the lockfile. So it seems like the addition of uuid in our stuff was not properly taken out. I think you could just do a `git checkout package-lock.json` to revert it.